### ISSUE TITLE
No CSS class to render tag "tr"/"tbody"

### ISSUE THREAD

**Issue by bstream** [sapegin/react-styleguidist#130]
Hi, I just started using react-styleguidist and I noticed that there are some missing classes for markdown parsed tables. The table renders fine but I got some hints in the console that I should report it here.

![console image](https://www.dropbox.com/s/yqqsf55i08f6ocl/Screenshot%202016-04-21%2009.01.21.png?dl=1)

### Example:

```md
**Table name**

| Column1   | Column2   | Column3   |
| :-------- | :-------- | :-------- |
| Content1  | Content2  | Content3  |
```
**Table name**

| Column1   | Column2   | Column3   |
| :-------- | :-------- | :-------- |
| Content1  | Content2  | Content3  |

**Comment by sapegin** [comment]
Thanks! That’s fine if you don’t want to change styles of this tags.
